schemata/pdf_manager_template.json

```diff
@@ -3,7 +3,6 @@
   "id": "/schemata/pdf_manager_template",
 
   "title": "PDF Manager Template",
-  "stability": "prototype",
```

Comment:
From [Heroku's API Compatibility Policy](https://devcenter.heroku.com/articles/api-compatibility-policy):

> There are three levels of stability: prototype, development, and production.

Seems like we should use `production`.